docs/features/backup_and_restore.md

```diff
@@ -6,7 +6,201 @@ hide_title: false
+
+##### Halt Ingest
+
+Deactivate all Cumulus Rules, halt all clients that access the archive API and
+stop any other database accessor processes.   Ensure all active executions have
+completed before proceeding.
+
+##### Remove Database Cluster Access
+
+Depending on your database cluster/setup, there are several ways to limit access to the
+database.   One example:
+
+Log in as the administrative user to your database cluster and run:
+
+```sql
+alter database my_database connection limit 0;
+select pg_terminate_backend(pg_stat_activity.pid) from pg_stat_activity where pg_stat_activity.datname = 'database';
+```
+
+This should block new connections to the Core database from the database user
+and cause database writes to fail.
+
+Take care that you aren't using the admin user (e.g. postgres) with a default
+database or you may inadvertently limit connections by your administrative
+tasks.
+
+#### **2. Using the AWS CLI (see [AWS PITR documentation](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_PIT.html) for console instructions), run the following command, making *certain* to use the same subnet groups and vpc-security-group IDs from your Core deployment:**
+
+  ```bash
+  aws rds restore-db-cluster-to-point-in-time --source-db-cluster-identifier "<cluster-needing-restoration>" --restore-to-time "<time>" --vpc-security-group-ids "<security-group-1>" "<security-group-2>" --copy-tags-to-snapshot --db-cluster-identifier "<new-cluster-identifier>" --db-subnet-group-name "<db-subnet-group>"
+  ```
+
+  You can get the configuration vales from the [RDS
+  console](https://console.aws.amazon.com/rds/) *or* by running the following
+  command and parsing the outputs:
+
+  ```bash
+  aws rds describe-db-clusters
+  ```
+
+* cluster-needing-restoration -- the name of the database cluster you're
+  restoring *from* (`DBClusterIdentifier` from the AWS RDS CLI output)
+* time - The time in UTC format (e.g. 2015-03-07T23:45:00Z)
+* security-group-# - the security group IDs from your original deployment
+* new-cluster-identifier - The cluster name for the backup replica.   This
+  *must* be different than the original
+* db-subnet-group - The db subnet group created for the original cluster
+  (`DBSubnetGroup` from the AWS RDS CLI output)
+
+  Once this command is run, you should see the cluster appear in the RDS cluster
+  list with a `Creating` status.  Verify the creating cluster has a
+  reasonable configuration.   Once the cluster is online, manually validate
+  that it has the tables/data you expect, then proceed.
+
+#### 3. Import cluster into terraform state
+
+Run the following commands to bring the new cluster into the
+terraform state file, where {module_name} is the title you've assigned to the module:
+
+* Remove the old cluster from your terraform state:
+
+```bash
+terraform state {module_name}.aws_rds_cluster.cumulus
```

Comment:
What would be the `module_name` for our example deployment? I'm finding it hard to confirm that this terraform resource path is correct.

Reply:
I believe that should be  `rds_cluster` as seen in https://github.com/nasa/cumulus/blob/master/example/rds-cluster-tf/main.tf